From: intrigeri <intrigeri@boum.org>
Date: Tue, 3 Jul 2018 15:43:10 +0000
Subject: AppArmor: allow Firefox to read
 /usr/share/glib-2.0/schemas/gschemas.compiled.

Otherwise, Tor Browser 8.0a9 crashes when clicking "Save Page As".

(cherry picked from commit ad95bbda69045f3c9ace241939ee9e1fccc16ac8)
---
 apparmor/torbrowser.Browser.firefox | 1 +
 1 file changed, 1 insertion(+)

diff --git a/apparmor/torbrowser.Browser.firefox b/apparmor/torbrowser.Browser.firefox
index d0aded9..c145d06 100644
--- a/apparmor/torbrowser.Browser.firefox
+++ b/apparmor/torbrowser.Browser.firefox
@@ -68,6 +68,7 @@ profile torbrowser_firefox @{torbrowser_firefox_executable} {
   /etc/mime.types r,
 
   /usr/share/ r,
+  /usr/share/glib-2.0/schemas/gschemas.compiled r,
   /usr/share/mime/ r,
   /usr/share/themes/ r,
   /usr/share/applications/** rk,
